connector/src/test/java/com/google/cloud/hive/bigquery/connector/acceptance/DataprocAcceptanceTestBase.java

```diff
@@ -32,7 +32,7 @@
 import static com.google.cloud.hive.bigquery.connector.acceptance.AcceptanceTestUtils.readGcsFile;
 import static com.google.cloud.hive.bigquery.connector.acceptance.AcceptanceTestUtils.uploadConnectorInitAction;
 import static com.google.cloud.hive.bigquery.connector.acceptance.AcceptanceTestUtils.uploadConnectorJar;
-import static com.google.common.truth.Truth.assertThat;
```

Comment:
Why the assert was replaced? please reduces the changes to the minimum

Reply:
I had removed Truth because it was only used in a couple places where it wasn't absolutely needed. So I thought it'd be an opportunity to reduce the amount of dependencies. But I'll revert it to keep the PR simple.